conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -421,12 +419,10 @@ msgid "Found existing group: \"{}\""
 msgstr ""
 
 #: common/djangoapps/student/management/commands/manage_group.py
-#, python-brace-format
```

Comment:
@e-kolpakov Do you have any idea why my `paver i18n_generate` removed these lines?  Is it something I can/should fix?